client/src/app/EmptyTab.js

```diff
@@ -39,12 +39,14 @@ export default class EmptyTab extends PureComponent {
       <Tab className={ css.EmptyTab }>
         <p className="create-buttons">
           <span>Create a </span>
-          <button className="create-bpmn btn btn-secondary" onClick={ () => onAction('create-bpmn-diagram') }>BPMN diagram</button>
```

Comment:
Seems like another component's tests depend on the `create-bpmn` selector: https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/__tests__/AppSpec.js#L265

Reply:
Sounds like an HoC snippet.

Reply:
Thanks for the hint.